src/Microsoft.Dnx.Tooling/Program.cs

```diff
@@ -80,7 +80,28 @@ public int Main(string[] args)
             WrapConsoleCommand.Register(app, reportsFactory);
             FeedsConsoleCommand.Register(app, reportsFactory);
 
-            return app.Execute(args);
+            try
+            {
+                return app.Execute(args);
+            }
+            catch (CommandParsingException ex)
+            {
+                AnsiConsole.GetError(_runtimeEnv.OperatingSystem == "Windows").WriteLine(ex.Message.Red().Bold());
```

Comment:
@anurse This was done to mimic ReportFactory.cs in tooling where ansiconsole was used. I thought that maybe color options were only available for windows environments?